### PR TITLE
LR.UI Add Config Page

### PR DESCRIPTION
**Pull request by alfredoihernandez**
# Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## PR Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code follows the code style of this project.
- [ ] I ran lint checks locally prior to submission.
- [ ] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Purpose of PR
Create New Config Page
Embed Config Form in Config Page
Modify Config Form to work in both modal and page
Add GetConfig to Config Service
Remove PendingFeature Modal from Config (use pencil icon to view/edit)

image

## Does this introduce a breaking change

- [ ] YES
- [x] NO

## Validation

- [ ] Unit tests updated and ran successfully
- [ ] Update documentation or issue referenced above

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/wcnp/issues/650

### CODE REVIEW

File: src/LodeRunner.UI/src/components/Configs/index.js [hunk 1]
```diff
@@ -142,19 +131,13 @@ const Configs = () => {
...
               </div>
               <div className="configs-item-options">
-                <button
-                  className="unset"
-                  type="button"
-                  onClick={openConfigFormModal(configId)}
-                  onKeyDown={openConfigFormModal(configId)}
-                  aria-label="Edit Load Test Config"
-                >
+                <A href={`/configs/${configId}`}>
```
Comment (jofultz) [inline src/LodeRunner.UI/src/components/Configs/index.js:134]: Should the anchor tag be lowercase \<a\/> ?

File: src/LodeRunner.UI/src/components/ConfigPage/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import "./styles.css";
+
+const ConfigPage = ({ configId }) => {
+  return (
+    <div className="config">
+      <div className="page-header">
+        <h1>
+          Load Test Config
+        </h1>
+        <A href="/" className="unset navigation">
```
Comment (jofultz) [inline src/LodeRunner.UI/src/components/ConfigPage/index.js:13]: Should anchor be lowercase?

**Review by jofultz — CHANGES_REQUESTED** [review]
Just a question about whether the anchor tags were intended to be capitalized or if they should be lowercase.

File: src/LodeRunner.UI/src/components/Configs/index.js (hunk 1 above)
Comment (alfredoihernandez) [reply src/LodeRunner.UI/src/components/Configs/index.js:134]: A comes from: import { A } from "hookrouter";

File: src/LodeRunner.UI/src/components/ConfigPage/index.js (hunk 2 above)
Comment (alfredoihernandez) [reply src/LodeRunner.UI/src/components/ConfigPage/index.js:13]: A comes from: import { A } from "hookrouter";

**Comment by alfredoihernandez** [comment]
> Just a question about whether the anchor tags were intended to be capitalized or if they should be lowercase. Also, some checks failed.

fix lint fix check

**Review by jofultz — APPROVED** [review]

**Review by joheec — APPROVED** [review]

**Pull request merged**